tools/cloudera/build_hue_cloudera.sh

```diff
@@ -46,6 +46,7 @@ function install_prerequisite() {
     export FORCEINSTALL=0
     export SQLITE3_PATH="$TOOLS_HOME/sqlite/bin/sqlite3"
     redhat7_ppc_install
+    # NODEJS install
     source /opt/rh/rh-nodejs14/enable
```

Comment:
This is still on 14, should be 16 right?

Reply:
Yes, it should be 16. Moved this part to build_hue_common.sh.